README.md

```diff
@@ -1,2 +1,129 @@
-# community
-CoreDash Governance
+# Welcome to the CoreDash Community!
+
+With the recent license changes of the Grafana project, there are quite a few
+community end users as well as monitoring vendors who are interested in a
+visualization and dashboarding solution that is under the Apache-2.0 
+license, owned by the Linux Foundation, and governed by a mix of community
+end users and vendors. The CoreDash Project is designed as a place where we can
+work together on this solution. Please note that CoreDash is not designed to be
+a fork of Grafana, but will instead take a more componetized approach as a
+collection of subprojects that users can select from to meet their needs.
+
+CoreDash is the current working name / code name for this effort, but we expect
+to rename it after we've made a bit more progress and better defined what we
+plan to do.
+
+## Getting Started
+
+### Communication
+
+Please join the coredash-discuss@googlegroups.com 
+[mailing list](https://groups.google.com/g/coredash-discuss)
+where we have discussions about the project.
+
+We meet on the 1st and 3rd Thursday of every month at 7am PT | 10am ET | 15:00 UK
+for 1 hour. Agendas, meeting notes, and joining details can be found in our
+[working session
+doc](https://docs.google.com/document/d/1FZy-tuIz-C5NSQe3AdeT-DixZZXMjDi-BnQy2kj_9xU/edit).
+ 
+Participation in this community is subject to our
+[Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Subprojects
+
+You can start now by contributing to one of our [subprojects](subprojects.md). 
+
+If you have a project (or want to start one) that would be a good fit for this
+effort, please reach out on the mailing list to discuss contributing a new
+subproject to this Linux Foundation effort.
+
+## Draft Scope
+
+**NOTE**: This scope section is still evolving and should be expected to change
+as we get more people and projects involved in this effort.
+
+If we look at the Observability and Analysis segment of the CNCF Landscape, the
+three projects in metrics based monitoring (Prometheus, Cortex and Thanos) are
+all Prometheus ingest format and PromQL based solutions. The other graduated or
+incubating projects are fluentd in logging which has no standard ingest protocol
+or query language yet and Jaeger and OpenTracing, which have a standard
```

Comment:
Jaeger is a storage system, OpenTracing is an instrumentation library. OpenTracing doesn't have a standard protocol, but a standard API surface for instrumentation libraries. On the other hand, OpenTelemetry is aiming to standardize the ingestion protocol for tracing.

We should say something like Jaeger has a custom ingestion protocol and custom querying / visualization / UX, and the community is standardizing the ingesting format with OpenTelemetry.

Reply:
@rakyll - I made this change and reworded that section a bit, since have one huge sentence about both Fluentd and Jaeger got unwieldy.  How does this look now?